### PR TITLE
nodereport project packaging updates

### PR DESCRIPTION
**Pull request by rnchamberlain**
Improvements and corrections to nodereport README, additional metadata in package.json and new MAINTAINER file to document NPM release procedure. See commits for details.

### CODE REVIEW

File: MAINTAINER.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# Instructions for maintainers of the nodereport project
+
+## Releasing to www.npmjs.com
```
Comment (sam-github) [inline MAINTAINER.md]: Are you sure? (some) People are already confused aboutthe difference between npm the cli, and npmjs.org the package repository... but above should be `npmjs.org`, IMO, not the company URL.

Comment (sam-github) [inline MAINTAINER.md]: Fair enough, "publish to npm" is clear enough. Anybody with the rights to publish nodereport should know what they are publishing to.

Comment (rnchamberlain) [inline MAINTAINER.md]: I just took the URL from my browser, npmjs.org re-directs immediately to npmjs.com. 

OK, I will change to 'publish to npm', people know what that means.

File: README.md [hunk 2]
```diff
@@ -56,10 +64,11 @@ Configuration on module initialisation is also available via environment variabl
...
-node_modules/nodereport/demo directory:
+To see examples of NodeReports generated from these events you can run the
+demonstration applications provided in the nodereport github repository. These are
+Node.js applications which will prompt you to trigger the required event.
 
-    api.js - NodeReport triggered by Javascript API call
-    exception.js - NodeReport triggered by unhandled exception
-    fatalerror.js - NodeReport triggered by fatal error on Javascript heap out of memory
-    loop.js - looping application, NodeReport triggered using kill -USR2 <pid>
+1. api.js - NodeReport triggered by Javascript API call
```
Comment (cjihrig) [inline README.md]: Javascript -> JavaScript here and elsewhere.

**Comment by richardlau** [comment]
Maybe update `AUTHORS` to match the contributors listed in `package.json`?

**Comment by sam-github** [comment]
Personally, I'd put a generic email address in package.json, and delete AUTHORS, its painful to maintain 2 copies of what is already available in git metadata. If not, at least script the generation of AUTHORs: https://github.com/sam-github/git-extras/blob/master/bin/git-authors

**Comment by rnchamberlain** [comment]
Looks like AUTHORS is a github thing, whereas the package.json property came originally from 'npm init'. Other nodejs projects vary, eg https://github.com/nodejs/llnode has both, and they are not consistent. OK, lets follow the https://github.com/nodejs/nan example, and just have the contributors array in package.json, deleting AUTHORS.

**Comment by rnchamberlain** [comment]
ping @richardlau @sam-github @cjihrig are you OK with this PR now? thanks

**Comment by sam-github** [comment]
LGTM

**Pull request merged**